src/django_upgrade/fixers/model_field_choices.py

```diff
@@ -0,0 +1,58 @@
+"""
+Drop `.choices` for model field `choices` parameters:
+https://docs.djangoproject.com/en/5.0/releases/5.0/#forms
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset
+from tokenize_rt import Token
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer
+from django_upgrade.data import State
+from django_upgrade.data import TokenFunc
+from django_upgrade.tokens import find_last_token
+
+fixer = Fixer(
+    __name__,
+    min_version=(5, 0),
+)
+
+
+@fixer.register(ast.Call)
+def visit_Call(
+    state: State,
+    node: ast.Call,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        not state.looks_like_migrations_file
+        and (
+            (
+                isinstance(node.func, ast.Attribute)
+                and "models" in state.from_imports["django.db"]
+                and isinstance(node.func.value, ast.Name)
+                and node.func.value.id == "models"
+            )
+            or (
+                isinstance(node.func, ast.Name)
+                and node.func.id in state.from_imports["django.db.models"]
+            )
+        )
```

Comment:
I think it's actually very limiting to only act on fields from `django.db.models`. Custom fields exist in many places, like `django.contrib.postgres`, third party apps, and even projects. Let's actually limit this to calls to things with a name ending in `Field` ? Need to handle names and chains of `Attribute`s like `django.db.models.CharField(...)`.